doc/nrf/libraries/bluetooth/cs_de.rst

```diff
@@ -0,0 +1,43 @@
+.. _cs_de_readme:
+
+Bluetooth Channel Sounding Distance Estimation
+##############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
```

Comment:
We could remove the "Overview" section as there is no intro sentence above overview.

Reply:
so just remove these two lines?

Reply:
yeah, line no:10 and 11.